src/core/src/core_logic/ExecutionConfig.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, env_layer, composite_logger, execution_parameters):
         self.included_classifications_list = self.__get_execution_configuration_value_safely(self.config_settings, Constants.ConfigSettings.CLASSIFICATIONS_TO_INCLUDE, [])
         self.included_package_name_mask_list = self.__get_execution_configuration_value_safely(self.config_settings, Constants.ConfigSettings.PATCHES_TO_INCLUDE, [])
         self.excluded_package_name_mask_list = self.__get_execution_configuration_value_safely(self.config_settings, Constants.ConfigSettings.PATCHES_TO_EXCLUDE, [])
+        self.max_patch_publish_date = self.__get_execution_configuration_value_safely(self.config_settings, Constants.ConfigSettings.MAX_PATCH_PUBLISH_DATE, [])
```

Comment:
The last parameter is the default value. Max patch publish date is not an array like the others -- ensure the code works correctly when this value is not specified (this will be a valid case when imperative goes out). Ensure the intended behavior in that case is also in the design doc (currently missing).

Reply:
Changed the default value to empty string. 
For Linux, max patch publish date is only used in status reporting, and that part will all scenarios are mentioned [here](https://microsoft.sharepoint.com/:w:/r/teams/GuestPatching/_layouts/15/Doc.aspx?sourcedoc=%7B3BB6D095-C8A3-4C5B-AC68-68F69FE3A0BC%7D&file=Patch%20Management%20V2%20-%20Extension%20Spec%20(OS-Agnostic).docx&action=default&mobileredirect=true), which is also linked in the design doc.